roles/openshift_logging_elasticsearch/defaults/main.yml

```diff
@@ -1,19 +1,44 @@
 ---
+
 ### Common settings
 openshift_logging_elasticsearch_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default('docker.io/openshift/origin-') }}"
+openshift_logging_elasticsearch_image_name: "logging-elasticsearch"
 openshift_logging_elasticsearch_image_version: "{{ openshift_hosted_logging_deployer_version | default('latest') }}"
 openshift_logging_elasticsearch_image_pull_secret: "{{ openshift_hosted_logging_image_pull_secret | default('') }}"
 openshift_logging_elasticsearch_namespace: logging
-
+openshift_logging_elasticsearch_sa_name: aggregated-logging-elasticsearch
+kubeconfig: /etc/origin/master/openshift-master.kubeconfig
 openshift_logging_elasticsearch_nodeselector: "{{ openshift_logging_es_nodeselector | default('') }}"
 openshift_logging_elasticsearch_cpu_limit: 1000m
 openshift_logging_elasticsearch_memory_limit: "{{ openshift_logging_es_memory_limit | default('1Gi') }}"
 openshift_logging_elasticsearch_recover_after_time: "{{ openshift_logging_es_recover_after_time | default('5m') }}"
+openshift_logging_elasticsearch_clusterreader: true
 
 openshift_logging_elasticsearch_replica_count: 1
 
 # ES deployment type
 openshift_logging_elasticsearch_deployment_type: "data-master"
+openshift_logging_elasticsearch_node_topology:
```

Comment:
In the inventory file people can provide JSON with this structure. is that acceptable?
i.e. smth like:
`openshift_logging_elasticsearch_node_topology='{"masters": {"replicas": 1, "limits": {"cpu": "200m"}...'`
i'll think on the translation...